### DISCUSSION TITLE
I have a issue with the speed and burn of my laser.

### DISCUSSION THREAD

**Discussion by Xander1983** [arkypita/LaserGRBL#905] [Q&A]
I am having an issue with the speed and burning of my laser. When I got it the stepper board wasn't working right so I got a new one. In the process I had downloaded other things to flash and update my arduino nano v2.5 board. After I got the new stepper motor it and put it in my board still wasnt acting right so I updated the firmware again and it worked perfectly. Just was burning to big of lines so I tried to fix the laser and in the process I went into lasergrbl and edited some setting that I saw a guy using online. Now my stepper motors are moving really fast and my laser just wants to create dots. I have no idea how to get it to slow back down or burn a straight line.
![feet](https://user-images.githubusercontent.com/58815956/76219788-c300b100-61ec-11ea-94c8-040adc260568.jpg)
![istockphoto-1043819012-1024x1024](https://user-images.githubusercontent.com/58815956/76219791-c3994780-61ec-11ea-8a30-fb587aef4f37.jpg)
![machine](https://user-images.githubusercontent.com/58815956/76219792-c431de00-61ec-11ea-8b3b-b2b6318d0036.jpg)
![settng](https://user-images.githubusercontent.com/58815956/76219794-c4ca7480-61ec-11ea-9c9c-21e67a6bdf78.jpg)

**Comment by arkypita** [comment]
$100 $101 set how many motor step is needed to move the head by 1mm
They look too high for me, usually they are in the order of 40-200 step/mm

The exact value depends by your hardware, Use stuart excel file to fine trim exact values
https://github.com/arkypita/LaserGRBL/issues/589

**Comment by StuartB4** [comment]
Also, I noticed you have $30, 31 and 32 in your grbl settings,  change $32 to 1 to enable Laser mode.
You can leave $30 set to 255.
